tests/testthat/test-expand.R

```diff
@@ -97,3 +97,17 @@ test_that("crossing checks for bad inputs", {
     "Problems: y"
   )
 })
+
+test_that("expand handles list columns", {
+  data_sets <- iris %>% split(iris$Species)
+  formulas <- list(formula1 = Sepal.Length ~ Sepal.Width,
+                   formula2 = Sepal.Length ~ Sepal.Width + Petal.Width,
+                   formula3 = Sepal.Length ~ Sepal.Width + Petal.Width + Petal.Length)
+  (demo <- tibble::tibble(formulas = formulas, data_sets = data_sets))
```

Comment:
I don't think we need the `tibble::` prefix here. Also, I'd suggest to create `demo` only once with a pipe instead of updating it, something like : 

```r
demo <- tibble(formulas = formulas, data_sets = data_sets)) %>%
  expand(formulas, data_sets)
```